### PR TITLE
fix: Prevent WatermelonDB clashing with other pods in RN0.62

### PR DESCRIPTION
**Pull request by district-michael**

### CODE REVIEW

File: native/ios/WatermelonDB.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -353,7 +353,7 @@
 					"$(SRCROOT)/../../../react-native/React/**",
 					"$(SRCROOT)/../../node_modules/react-native/Libraries/**",
 					"$(SRCROOT)/../../../../node_modules/react-native/Libraries/**",
-					"$(SRCROOT)/../../../../../ios/Pods/Headers/Public/**",
+					"$(SRCROOT)/../../../../../ios/Pods/Headers/Public/React-Core/**",
```
Comment (radex) [inline native/ios/WatermelonDB.xcodeproj/project.pbxproj:356]: could you explain your rationale behind this change?

Comment (district-michael) [reply native/ios/WatermelonDB.xcodeproj/project.pbxproj:356]: This fixes an issue where Watermelon fails to build on projects using Flipper, as there appears to be a conflict with the version of Folly that is bundled with it in react native 0.62.  As Watermelon only relies on React-Core, it doesn't need to search the entire pods folder recursively.

Comment (district-michael) [reply native/ios/WatermelonDB.xcodeproj/project.pbxproj:356]: See the discussion in this issue for more info:
https://github.com/Nozbe/WatermelonDB/issues/655

Comment (district-michael) [reply native/ios/WatermelonDB.xcodeproj/project.pbxproj:356]: @radex Do you have any comments on this?

Comment (brian-perlego) [reply native/ios/WatermelonDB.xcodeproj/project.pbxproj:356]: This PR could unblock my team, any update @radex ?

**Comment by brian-perlego** [comment]
Why not changing it here @district-michael 
https://github.com/Nozbe/WatermelonDB/blob/00d56201fab00e29fee8dc3d253f623e582bb004/native/ios/WatermelonDB.xcodeproj/project.pbxproj#L407

**Comment by brian-perlego** [comment]
Could you also change it for Release build ?
https://github.com/Nozbe/WatermelonDB/blob/00d56201fab00e29fee8dc3d253f623e582bb004/native/ios/WatermelonDB.xcodeproj/project.pbxproj#L407

**Comment by district-michael** [comment]
@brian-perlego I didn't change for release because Flipper _should't_ be included in release builds, but I guess there is no harm in changing for release too, as that may provide for a more consistent experience.

File: native/ios/WatermelonDB.xcodeproj/project.pbxproj (hunk 1 above)
Comment (radex) [reply native/ios/WatermelonDB.xcodeproj/project.pbxproj:356]: @district-michael And doesn't this change break Watermelon for RN < 0.62?

Comment (district-michael) [reply native/ios/WatermelonDB.xcodeproj/project.pbxproj:356]: @radex I just tested in our project on 0.61 prior to our upgrade and it works fine.  Don't have any code prior to that version to test on.

Comment (radex) [reply native/ios/WatermelonDB.xcodeproj/project.pbxproj:356]: OK, cool! I'll test this on my project, and hopefully merge sometime soon :)

Comment (davidsonsns) [reply native/ios/WatermelonDB.xcodeproj/project.pbxproj:356]: I had the same issue. Just spent a whole day looking into it. Any plans to merge it?
FYI:
```js
"@nozbe/watermelondb": "^0.16.0",
"react": "16.11.0",
"react-native": "0.62.2",
```

**Pull request merged**